ts/nixpkgs.ts

```diff
@@ -0,0 +1 @@
+export * from "./internal/nixpkgs/mod.ts";
```

Comment:
This will cause type errors locally until running codegen. Should we ignore this file again, or check in the codegen files?

Reply:
Ignoring is not really an option, since we need this file in the repo and it's hand-written, right?

We could generate this file too. Just `writeFile "ts/nixpkgs.ts" "export * from ...\n"`.

Is there a real downside to checking in the generated files though? Maybe we need a CI check that they're in sync then. But I wouldn't be against that. (But we could defer that to another PR.)

Reply:
Yeah, I think that makes sense.